### DISCUSSION TITLE
How to build in Neper

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#609] [Q&A] [answered]
Hello Respected Professor Dr. Romain Quey,
Can I build this type of Model in Neper?

Thank You
![5741](https://user-images.githubusercontent.com/102291665/202492292-e506ba4f-15c8-48f1-b4a6-3bf352915e28.png)

**Comment by rquey** [comment, marked as answer]
This is not a polycrystal, so, no. (At best, you can describe it by a tesr.)

**Comment by Ali141272** [comment]
Thank you for your kind response.
    On Friday, November 18, 2022 at 12:41:54 AM GMT+8, Romain Quey ***@***.***> wrote:  
 
 


This is not a polycrystal, so, no.
